### PR TITLE
DG: update appendix: effort

### LINKED ISSUE #256: Undo enter current session bug (Major)
Steps:
1) enterses and use presence 5 (both succeed)
2) undo presence 5(succeed)
3) undo enterses (succeed)
4) Redo enterses 
result:
- UI shows that im in current session but cannot use presence 5 
image
- state is also in session 5
image

### PR DESCRIPTION
**Pull request by erisjacey**
- Also update PPP for @erisjacey 
- Also fixes bug in issue #256

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1441,24 +1441,54 @@ Deleting sessions in the students tab
...
 
-{ start of `appendix` written by: _________ }
+{ start of `appendix` written by: Masagca Eris Jacey }
 
 ## 12. Appendix: effort
 
 **Difficulty level:**
 
-* todo
+The overall difficulty level of the project was relatively moderate.
```
Comment (CodingCookieRookie) [inline docs/DeveloperGuide.md]: Should this be relatively difficult? :DDD

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1441,24 +1441,54 @@ Deleting sessions in the students tab
...
+   * "undo/redo": This was a feature that was hard to get right because of the different entity types being slightly different in nature.
+     ("students" were a "one-layer" entity while each "session" had their own list of "attributes", making it a "two-layer" entity)
 
 **Effort required:**
 
-* todo
+* Weekly meetings were held (about 2 hours every Wednesday) to review what had been done in the week prior, as well as 
+  discuss future tasks for the next week.
+* Our team worked tirelessly week in and week out for several (~7) weeks, sparing no effort during each iteration 
+  (on both new implementations as well as making sure the older ones aren't buggy).
```
Comment (CodingCookieRookie) [inline docs/DeveloperGuide.md:1474]: Minor nitpick but should aren't be are not? And emphasized -> emphasised, noticed in other areas as well

**Review by CodingCookieRookie — APPROVED** [review]
LGTM with very minor nitpicks

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (erisjacey) [reply docs/DeveloperGuide.md:1474]: I think emphasize is the more common version, at least in this module we seem to be following American conventions (color, etc.)

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (erisjacey) [reply docs/DeveloperGuide.md]: Will be changing to that

**Pull request merged**